### ISSUE TITLE
Please archive this project rather than fixing <Link> issue

### ISSUE THREAD

**Issue by sorphwer** [vercel/mongodb-starter#28]
TL;DR: The `package.json` should not be updated to Next 13, while the entire project is built with Next 12. 

For anyone who wants to deploy this project:
- Use next 12.1.6 all the time, just like the lock file suggests. If you just clone the main branch and do `npm install` you will get Link tag error

For Vercel team:
- I suggest you to archive this project or fix `package.json`

More information: I notice that [gazdagergo](https://github.com/gazdagergo)  and [hkbertoson](https://github.com/hkbertoson) created PR to fix Link tag error, and I tried your fix but the error still exists. 

Regards,

**Comment by blaiseshema17** [comment]
New repository
